### DISCUSSION TITLE
Globe sync with new MapLibre globe?

### DISCUSSION THREAD

**Discussion by jatech1** [visgl/deck.gl#9254] [Q&A]
Has anyone found a way to use MapLibre globe mode along with deck.gl layers?  Any way to match the globe radius and fully sync?

**Comment by vatoutou** [comment]
Same problem here

**Comment by vatoutou** [comment]
Unable to synchronise deck.gl (arc & geojson layers) with maplibre v5 globe projection.


Here is the video 👇 
https://github.com/user-attachments/assets/56adef37-d513-4a6b-b207-334f9210ce7d


Tried to use `@vis.gl/react-maplibre` instead of `react-map-gl`,
Tried to add a `GlobeView` in the deck.gl overlay as well, that's just adding a new layer.
Tried to update all deck.gl, maplibre to latest & next vesions
Tried to wrap my maplibre component in `<DeckGL>' to let DeckGL control the map.

Nothing worked.

```json
    "react": "^18.3.1",
    "react-dom": "^18.3.1"
    "@deck.gl/core": "^9.0.40",
    "@deck.gl/layers": "^9.0.40",
    "@deck.gl/mapbox": "^9.0.40",
    "@deck.gl/react": "^9.0.40",
    "deck.gl": "^9.0.40",
    
    "@vis.gl/react-maplibre": "^1.0.0-alpha.5",
```

I found examples of using deck.gl with maplibre v5 globe
https://github.com/visgl/deck.gl/pull/9201
https://github.com/visgl/deck.gl/blob/master/examples/website/maplibre/app.tsx

None of them work.

**Reply by jatech1** [reply]
Have you tried deck 9.1 beta?  I believe that has the updated necessary to sync globe with maplibre 5.

**Reply by vatoutou** [reply]
I thought I did, but tried again with `"deck.gl: "^9.1.0-beta.3",` and it worked, thanks bro.

**Reply by jatech1** [reply]
Good to hear.  I had not had a chance to try it yet.  Now, just curious about intended release date for 9.1.

**Reply by vatoutou** [reply]
It feel like there is still some issues with interleaved layers. Maybe gonna wait a bit before using globe projection with deckgl 👀 

https://github.com/user-attachments/assets/aee9887e-a1ab-4afb-8a1b-0862be2918c4
